bukuserver/forms.py

```diff
@@ -1,44 +1,44 @@
-        raise ValidationError('Tag must not contain delimiter \"{}\".'.format(DELIM))
+        raise ValidationError(_('Tag must not contain delimiter "%(delim)s".', delim=DELIM))
 
 
 class SearchBookmarksForm(FlaskForm):
-    keywords = FieldList(StringField('Keywords'), min_entries=1)
-    all_keywords = BooleanField('Match all keywords', default=True, description='Exclude partial matches (with multiple keywords)')
-    markers = BooleanField('With markers', default=True, description=dedent('''\
-        The search string will be split into multiple keywords, each will be applied to a field based on prefix:
-         - keywords starting with '.', '>' or ':' will be searched for in title, description and URL respectively
-         - '#' will be searched for in tags (comma-separated, partial matches; not affected by Deep Search)
-         - '#,' is the same but will match FULL tags only
-         - '*' will be searched for in all fields (this prefix can be omitted in the 1st keyword)
-        Keywords need to be separated by placing spaces before the prefix.
-    '''))
-    deep = BooleanField('Deep search', description='When unset, only FULL words will be matched.')
-    regex = BooleanField('Regex', description='The keyword(s) are regular expressions (overrides other options).')
+    keywords = FieldList(StringField(_l('Keywords')), min_entries=1)
+    all_keywords = BooleanField(_l('Match all keywords'), default=True, description=_l('Exclude partial matches (with multiple keywords)'))
+    markers = BooleanField(_l('With markers'), default=True, description=LazyString(lambda: '\n'.join([
+        _('The search string will be split into multiple keywords, each will be applied to a field based on prefix:'),
+        _(" - keywords starting with '.', '>' or ':' will be searched for in title, description and URL respectively"),
+        _(" - '#' will be searched for in tags (comma-separated, partial matches; not affected by Deep Search)"),
+        _(" - '#,' is the same but will match FULL tags only"),
+        _(" - '*' will be searched for in all fields (this prefix can be omitted in the 1st keyword)"),
+        _('Keywords need to be separated by placing spaces before the prefix.'),
+    ])))
+    deep = BooleanField(_l('Deep search'), description=_l('When unset, only FULL words will be matched.'))
+    regex = BooleanField(_l('Regex'), description=_l('The keyword(s) are regular expressions (overrides other options).'))
 
 
 class HomeForm(SearchBookmarksForm):
-    keyword = StringField('Keyword')
+    keyword = StringField(_l('Keyword'))
 
 
 class BookmarkForm(FlaskForm):
-    url = StringField('Url', name='link', validators=[InputRequired()])
-    title = StringField()
-    tags = StringField()
-    description = TextAreaField()
+    url = StringField(_l('URL'), name='link', validators=[InputRequired()])
+    title = StringField(_l('Title'))
+    tags = StringField(_l('Tags'))
+    description = TextAreaField(_l('Description'))
```

Comment:
The names need to be provided explicitly, both to detect them when generating translation files and to apply translation to the field name.